caravel/assets/javascripts/explorev2/components/TextField.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { PropTypes } from 'react';
+import { FormGroup, FormControl } from 'react-bootstrap';
+import ControlLabelWithTooltip from './ControlLabelWithTooltip';
+
+const propTypes = {
+  label: PropTypes.string,
+  description: PropTypes.string,
+};
+
+const defaultProps = {
+  label: null,
+  description: null,
+};
+
+export default function TextField({ label, description }) {
+  return (
+    <FormGroup controlId="formInlineName">
```

Comment:
All of the `.*Field` component seem to repeat the `<FormGroup>< ControlLabelWithTooltip >` pattern, maybe this should live in `FieldSet` (aka FieldFactory)? Or maybe they inject their FormControl as a children of a `FieldWrapper`